posthog/api/decide.py

```diff
@@ -285,6 +285,14 @@ def get_decide(request: HttpRequest):
                         }
                     )
 
+                    mask_all_inputs = team.session_replay_config.get("mask_all_inputs", None)
+                    if mask_all_inputs is not None:
+                        session_recording_response["maskAllInputs"] = mask_all_inputs
+
+                    mask_all_text = team.session_replay_config.get("mask_all_text")
+                    if mask_all_text is not None:
+                        session_recording_response["maskTextSelector"] = "*" if mask_all_text else None
```

Comment:
only exposing mask _all_ text in settings now, but send it to the client as `maskTextSelector` since then we could let people set the selectors in remote settings in future